src/shader_compiler.cc

```diff
@@ -241,4 +252,68 @@ Result ShaderCompiler::CompileOpenCLC(const Shader*,
 }
 #endif  // AMBER_ENABLE_CLSPV
 
+Result ParseSpvEnv(const std::string& spv_env,
+                   uint32_t* target_env,
+                   uint32_t* target_env_version,
+                   uint32_t* spirv_version) {
+  if (!target_env || !target_env_version || !spirv_version)
+    return Result("ParseSpvEnv: null pointer parameter");
+
+  const uint32_t vulkan = 0;
```

Comment:
Move all these to the anonymous namespace and make constants.

Reply:
done